common-tools/clas-detector/src/main/java/org/jlab/detector/decode/CLASDecoder4.java

```diff
@@ -393,16 +420,24 @@ public Event getDataEvent(){
 
         Event event = new Event();
 
```

Comment:
Don't you want/need to add ATOF here?

Reply:
No. I don't use the standard TDC structure since we have a time-over-threshold to report. I added it to the `Dgz` class since it is transient but fill it in the `getDataBankTDCPetiroc` function.